src/main/resources/db/migration/V14__create_spm_svar_tables.sql

```diff
@@ -0,0 +1,16 @@
+CREATE TABLE sporsmal
+(
+    id      VARCHAR(64)   NOT NULL PRIMARY KEY,
```

Comment:
Skal Iden her være en UUID?

Reply:
eller skal det bare være en sekvens?

Reply:
Det holder kanskje at det er en sekvens? Vi kan kanskje bruke sekvens som id for svar også?